.github/workflows/Ontology_Mapper.yml

```diff
@@ -20,7 +20,6 @@ jobs:
 
     - name: Install Dependencies with Poetry
       run: |
-        keyring --disable
         poetry config http-basic.fury ${{ secrets.FURY_READ_TOKEN  }} NOPASS
```

Comment:
@jbkoh The token doesn't appear to be included in the command during the build process. Does this repository need special access to these tokens, or is there something else I'm overlooking?